### ISSUE TITLE
4.6.20 Make Pattern on Revit 2019.2

### ISSUE THREAD

**Issue by mpwuzhere** [eirannejad/pyRevit#596]
When exporting a pattern file to file and selecting "Inches" and "Drafting" it exports in "MM" and "MODEL".

**Expected behavior**
PAT file should be in Inches and Drafting

**Desktop (please complete the following information):**
 - OS: Windows 10
 - pyRevit Version 4.6.20
 - pyRevit Environment:  "==> Registered Clones (full git repos)

```
==> Registered Clones (deployed from archive)
master | Deploy: "basepublic" | Branch: "master" | Version: "4.6.20" | Path: "C:\Users\mpatrick\AppData\Roaming\pyRevit-Master"
==> Attachments
Autodesk Revit 2017.2.4 | Clone: "master" | Engine: "277"
Autodesk Revit 2018.3.1 | Clone: "master" | Engine: "277"
Autodesk Revit 2019.2.1 | Clone: "master" | Engine: "277"
Autodesk Revit 2020 First Customer Ship | Clone: "master" | Engine: "277"
==> Installed UI Extensions
==> Installed Library Extensions
==> Default Extension Search Path
C:\Users\mpatrick\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2017.2.4 | Version: 17.0.1158.0 | Build: 20181011_1645(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2017"
Autodesk Revit 2018.3.1 | Version: 18.3.1.2 | Build: 20180423_1000(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018"
Autodesk Revit 2019.2.1 | Version: 19.2.10.7 | Build: 20190225_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019"
Autodesk Revit 2020 First Customer Ship | Version: 20.0.0.377 | Build: 20190327_2315(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2020"
==> Running Revit Instances
PID: 11384 | Autodesk Revit 2019.2.1 | Version: 19.2.10.7 | Build: 20190225_1515(x64) | Language: 0 | Path: "C:\Program Files\Autodesk\Revit 2019"
==> User Environment
Microsoft Windows 10 [Version 10.0.17763]
Executing User: WAG\mpatrick
Active User: WAG\mpatrick
Adming Access: No
%APPDATA%: "C:\Users\mpatrick\AppData\Roaming"
Latest Installed .Net Framework: "4.7.2"
No .Net Target Packs are installed.
pyRevit CLI 0.9.8.0"
```

**Comment by eirannejad** [comment]
Okay fixed the issue with the incorrect type. I can not replicate the MM vs INCH issue. Would you share an example model?

**Comment by mpwuzhere** [comment]
Here ya go....also I get a bunch of warnings that normally are never seen.

[TESTRIGID.zip](https://github.com/eirannejad/pyRevit/files/3184801/TESTRIGID.zip)

[Rigidpat.zip](https://github.com/eirannejad/pyRevit/files/3184802/Rigidpat.zip)
![PyRevit_warning](https://user-images.githubusercontent.com/42358510/57816370-eaebc900-772f-11e9-8582-4a2761664f64.png)

**Comment by eirannejad** [comment]
Hey I can not replicate this error. Need a bit more info.

![2019-05-17_09-09-17](https://user-images.githubusercontent.com/8197916/57941366-99ece980-7883-11e9-8d41-2799e4722c74.gif)

**Comment by mpwuzhere** [comment]
Export PAT file, not Create Pattern

I tried it again from an OOTB template and it still saves my drafting pattern in inches to MM and Model.
Works fine if I keep it internally in the project.

**Comment by eirannejad** [comment]
Okay fixed this and will push with 4.6.21
